src/login/LoginPage.jsx

```diff
@@ -43,23 +50,40 @@ const LoginPage = (props) => {
     loginErrorContext,
     loginResult,
     shouldBackupState,
+    thirdPartyAuthContext,
```

Comment:
then we can remove lines 72, 73, 74 that are destructuring `thirdParyAuthContext`
```suggestion
   thirdPartyAuthContext: {
      providers,
      currentProvider,
      secondaryProviders,
      finishAuthUrl,
      platformName,
      errorMessage: thirdPartyErrorMessage,
    },
```